ui/src/components/forms/new-account.js

```diff
@@ -4,10 +4,11 @@ import { Button, Label, Spinner, TextInput } from 'flowbite-react';
 import useAddAccount from '~/hooks/add-account';
 
 const NewAccountForm = ({ setOpenModal }) => {
-  const { trigger } = useAddAccount();
+  const { mutate } = useAddAccount();
 
   const [error, setError] = useState('');
   const [form, setForm] = useState({
+    id: '0909090908',
```

Comment:
Why are we hard-coding this id? We should make that editable in the form and not provide a default.